programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,22 @@
+function solution(s){
+    s = s.toLowerCase();
+    let p = s.match(/p/g);
+    let y = s.match(/y/g);
```

Comment:
최초 선언 후에 값이 변하지 않으면 const로 선언하는게 좋습니다.
```suggestion
    const p = s.match(/p/g);
    const y = s.match(/y/g);
```